Moviebuff/Views/MovieListView.swift

```diff
@@ -0,0 +1,75 @@
+
+import SwiftUI
+
+struct MovieListView: View {
+    @ObservedObject private var nowPlayingState = MovieViewModel()
+    @ObservedObject private var bestDramasState = MovieViewModel()
+    @ObservedObject private var topRatedState = MovieViewModel()
+    @ObservedObject private var popularState = MovieViewModel()
+    
+    var body: some View {
+        NavigationView {
+            List {
+                Group{
+                    if nowPlayingState.movies != nil {
+                        PosterCarouselView(title: "Now Playing", movies: nowPlayingState.movies!, viewModel: nowPlayingState)
```

Comment:
There are a lot of force-unwraps here. As I mentioned elsewhere you can use `if let movies = nowPlayingState.movies` instead of doing this, as it's more resilient to refactoring and human errors.

Another approach would be to make your `var movies:[Movie]?` non-optional, but instead empty by default. Then if you need to check if you should display something, you could use `movies.isEmpty`.